app/components/Onboarding/NewWalletPassword.js

```diff
@@ -35,6 +36,9 @@ const NewWalletPassword = ({
       >
         Passwords do not match
       </p>
+      <p className={`${styles.helpMessage} ${passwordMinChars && styles.visible}`}>
```

Comment:
this visible class doesn't apply to `helpMessage` right? It looks like it's nested inside `errorMessage` in the `scss` file.

Also this logic seems to apply `visible` when the passwords match and the char count is 8 or over. Shouldn't it be the other way around?

Reply:
When I'm testing this PR the error message shows up before I start typing anything

Reply:
The help message is intended to always stay visible for better usability. I've just forgotten to clean it up the code.
If the help message is hidden some users will try to create passwords with less than 8 chars and then fail. They will be forced to rethink and resubmit their pass which is considerable user effort. We can prevent such cases by leaving the help message always visible. See [this article](https://www.nngroup.com/articles/password-creation/).